main/Checkbox/README.md

```diff
@@ -0,0 +1,52 @@
+# Check Box
+
+> Simple check box for react-native.
+> Refer : <https://material-ui.com/components/checkboxes>
+
```

Comment:
I recommend you to make a separate PR for docs work that is not related to this PR.

Reply:
I didn't think about this, but I found out thanks to you ✨ I'll reflect it in the next commit